pkg/k8shandler/common.go

```diff
@@ -584,6 +584,14 @@ func newVolumeSource(clusterName, nodeName, namespace string, node api.Elasticse
 		return volSource
 	}
 
+	// this should be caught by crd validation
+	// in case it somehow gets through be able to handle it gracefully
```

Comment:
Actually this can never happen.

Reply:
true, but nothing wrong with a little redundancy in case someone messes with their crd ;)